### ISSUE TITLE
bug: Can't pickup Questions on Mission Terminal

### ISSUE THREAD

**Issue by derFalke** [BosslandGmbH/BuddyWing#62]
doesn't work:

PickUpQuest QuestName="Emergency Measures" QuestId="0xE0000AE4D30908E0" Type="Placeable" UseRange="2" GiverName="Mission Terminal" X="-148.4512" Y="26.98392" Z="-32.84198"

**Comment by aevitas** [comment]
Does it give you the "You are not eligible" red error message?

**Comment by derFalke** [comment]
no, it stands only in front of the terminal and is doing nothing

**Comment by aevitas** [comment]
I'll check this out - thanks for reporting!
